src/libs/Navigation/AppNavigator/PublicScreens.js

```diff
@@ -7,7 +7,7 @@ import LogInWithShortLivedAuthTokenPage from '../../../pages/LogInWithShortLived
 import SCREENS from '../../../SCREENS';
 import defaultScreenOptions from './defaultScreenOptions';
 import UnlinkLoginPage from '../../../pages/UnlinkLoginPage';
-import AppleSignInDesktopPage from '../../../pages/signin/AppleSignInDesktopPage'
+import AppleSignInDesktopPage from '../../../pages/signin/AppleSignInDesktopPage';
```

Comment:
Thank you for cleaning up formatting issues! Can we keep changes where a file only has formatting changes in separate commits from logic changes, if not separate PRs?

Reply:
I'll remove it from this PR